static/javascript/launch.js

```diff
@@ -198,58 +202,59 @@ function setLaunchType(launchType) {
   }
 }
 
-function showSupplementaryData(show) {
+function showMetadataAccordion(type, show) {
+  let accordionElement = document.querySelector(
+    "#" + type + "-metadata-accordion",
```

Comment:
V minor/cosmetic - personal preference (so feel free to ignore!), but could use template literal / backticks here like `#${type}-metadata-accordion` like in the other places?

Reply:
Updated 👍